hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -3284,45 +3215,7 @@ protected boolean closeRegion(String encodedName, final boolean abort, final Ser
     return true;
   }
 
-  /**
-   * Close and offline the region for split or merge
-   *
-   * @param regionEncodedName the name of the region(s) to close
-   * @return true if closed the region successfully.
-   * @throws IOException
-  */
-  protected boolean closeAndOfflineRegionForSplitOrMerge(final List<String> regionEncodedName)
```

Comment:
Not used? Moved?

Reply:
Static analysis says it's not used. deleted.